### PR TITLE
Contract not declared error + Execution failed error support

### PR DESCRIPTION
**Pull request by Serhii-F-Braavos**

### CODE REVIEW

File: scripts/validator.js [hunk 1]
```diff
@@ -102,16 +102,18 @@ const validateErrorMatchers = (errorMatchers, _messageBase) => {
...
 
-            errorMatcher.extractors.forEach((extractor, extractorIdx) => {
-                try {
-                    stringToRegExp(extractor.matcher);
-                } catch (e) {
-                    errorsArray.push(
-                        errorMessageBase +
-                            `/extractors/${extractorIdx}/matcher contains invalid regExp`
-                    );
+            (errorMatcher.extractors || []).forEach(
```
Comment (avimak) [inline scripts/validator.js:105]:
```suggestion
            errorMatcher.extractors?.forEach(
```

File: tests/index.test.ts [hunk 2]
```diff
@@ -200,6 +191,14 @@ describe("formatError", () => {
                 result: "Error message: Oops: something is wrong",
             });
         });
+
+        it("if the error message doesn't contain contract address", () => {
+            expect(formatError(formatErrorTestData[11])).toEqual({
+                protocol: undefined,
+                interface: undefined,
+                result: "Error message default: Oops: something is wrong",
```
Comment (avimak) [inline tests/index.test.ts:199]: I'm not sure about the `Error message **default**` prefix

File: scripts/validator.js (hunk 1 above)
Comment (Serhii-F-Braavos) [reply scripts/validator.js]: done

File: tests/index.test.ts (hunk 2 above)
Comment (Serhii-F-Braavos) [reply tests/index.test.ts:199]: This message is only used in the tests configuration json file to make sure this case is handled by the default handler. Real users will never see this message. In production, different JSON file is used.

**Pull request merged**